app/views/candidate_mailer/application_withdrawn_on_request_offers_only.text.erb

```diff
@@ -4,7 +4,7 @@ Dear <%= @application_form.first_name %>,
 
 At your request, <%= @course.provider.name %> has withdrawn your application to study <%= @course.name_and_code %>.
 
-If you did not ask them to withdraw the application, contact a Get Into Teaching adviser:
+Get in touch if you did not ask them to do this:
```

Comment:
I've asked @EmmaFrith about this separately. I wonder if this is the right thing to point to in this circumstance.